app/services/authentication/authenticator.rb

```diff
@@ -88,7 +88,8 @@ def guard_against_spam_from!(identity:)
       return unless domain
       return if Settings::Authentication.acceptable_domain?(domain: domain)
 
-      message = "This #{identity.provider} email address #{identity.email} has been marked as spam"
+      message = "Sorry, but the domain for your email address is not allowed. This Forem may have limited signup to only
```

Comment:
Do we want to localize this message, instead of adding more English only copy to the code?

Reply:
Let's get this out the door then look to localize

Reply:
I agree with both of you. We should definitely localize this, but I just wanted to quickly fix it and I'm not sure if I can introduce new localized strings (haven't done it before) but I'm happy to try in followup PR.